merlin-server/src/main/java/gov/nasa/jpl/aerie/merlin/server/services/LocalMissionModelService.java

```diff
@@ -141,10 +155,10 @@ public Map<String, SerializedValue> getActivityEffectiveArguments(final String m
          InvalidArgumentsException
   {
     try {
-      return this.loadConfiguredMissionModel(missionModelId)
-                 .getActivityEffectiveArguments(activity.getTypeName(), activity.getArguments());
-    } catch (final MissionModelFacade.NoSuchActivityTypeException ex) {
-      throw new NoSuchActivityTypeException(activity.getTypeName(), ex);
+      return this.loadUnconfiguredMissionModel(missionModelId)
+                 .getActivityEffectiveArguments(activity);
```

Comment:
`MissionModelFacade#getActivityEffectiveArguments` is no longer used (except by other dead code starting at `validateActivityInstantiations`). Can we remove it?

Reply:
Method has been removed